packages/core/src/components/reviews/ReviewList/ReviewList.tsx

```diff
@@ -0,0 +1,186 @@
+  { ssr: false }
+)
+
+export interface ReviewListProps extends HTMLAttributes<HTMLDivElement> {
+  sortSelect?: {
+    label: string
+    options: Record<StoreProductListReviewsSort, string>
+  }
+  filterSelect?: {
+    label: string
+    options: Record<FilterProductListReview, string>
+  }
+  emptyList?: {
+    title: string
+    subtitle: string
+    buttonLabel: string
+  }
+  emptyFilter?: {
+    title: string
+    subtitle: string
+    buttonLabel: string
+  }
+  loadMoreLabel?: string
+  productId: string
+}
+
+function ReviewList({
+  sortSelect,
+  filterSelect,
+  emptyFilter,
+  emptyList,
+  productId,
+  loadMoreLabel = 'Load more',
+  ...props
+}: ReviewListProps) {
+  const { openReviewModal } = useUI()
+  const {
+    reviews,
+    loading,
+    loadingMore,
+    hasMore,
+    currentSort,
+    currentFilter,
+    loadMore,
+    onSortChange,
+    onFilterChange,
+    resetFilter,
+  } = useProductReviews({
+    productId: productId,
+  })
+
+  const isFirstLoading = loading && !loadingMore
+
+  if (!loading && reviews.length <= 0 && currentFilter === 'all') {
+    return (
+      <EmptyReviewList
+        {...props}
+        productId={productId}
+        onButtonClick={openReviewModal}
+        title={emptyList?.title}
+        subtitle={emptyList?.subtitle}
+        buttonLabel={emptyList?.buttonLabel}
+      />
+    )
+  }
+
+  return (
+    <div data-fs-review-list {...props}>
+      <div data-fs-review-list-header>
+        <SortProductReviews
+          id="fs-sort-product-reviews-select"
+          value={currentSort}
+          label={sortSelect?.label}
+          options={sortSelect?.options}
+          onChange={onSortChange}
+        />
+        <FilterProductReviews
+          id="fs-filter-product-reviews-select"
+          value={currentFilter}
+          label={filterSelect?.label}
+          options={filterSelect?.options}
+          onChange={onFilterChange}
+        />
+      </div>
+      <div data-fs-review-list-content>
+        <UIList className="review-list-items">
+          {isFirstLoading ? (
+            <ReviewCardSkeleton />
+          ) : reviews.length > 0 ? (
+            reviews.map((review) => (
+              <li key={review.id} className="review-list-item">
+                <UIReviewCard
+                  title={review.title}
+                  text={review.text}
+                  rating={review.rating}
+                  author={review.reviewerName}
+                  isVerified={review.verifiedPurchaser}
+                  date={new Date(review.reviewDateTime)}
+                />
+              </li>
+            ))
+          ) : (
+            <EmptyReviewList
+              data-fs-empty-review-filter
+              productId={productId}
+              title={emptyFilter?.title}
+              subtitle={emptyFilter?.subtitle}
+              buttonLabel={emptyFilter?.buttonLabel}
+              onButtonClick={resetFilter}
+            />
+          )}
+        </UIList>
+      </div>
```

Comment:
```suggestion
  
      { isFirstLoading ? (
        <ReviewCardSkeleton />) : (
          reviews.length > 0 ? (
            <UIList className="review-list-items">
              {reviews.map((review) => (
                <li key={review.id} className="review-list-item">
                  <UIReviewCard
                    title={review.title}
                    text={review.text}
                    rating={review.rating}
                    author={review.reviewerName}
                    isVerified={review.verifiedPurchaser}
                    date={new Date(review.reviewDateTime)}
                  />
                </li>
              ))}
            </UIList>
          ):(
            <EmptyReviewList
            data-fs-empty-review-filter
            productId={productId}
            title={emptyFilter?.title}
            subtitle={emptyFilter?.subtitle}
            buttonLabel={emptyFilter?.buttonLabel}
            onButtonClick={resetFilter}
            />
          )
        )
      }
```
avoid rendering the `EmptyReviewList` inside the list.

Reply:
@artursantiago if you accept the suggestion above just remember to check for the blank spaces to be according with linter format locally.

Reply:
Done in https://github.com/vtex/faststore/pull/2724/commits/97ff83407748e5b837bed65aefcd82376781e46e